declaration-diagnosis-redesign/specifying-model.qmd

```diff
@@ -61,94 +61,94 @@ Here, we have specified the details of the function that relates $Z$ and $U$ to
 
-A separate goal is assessing the performance of a research design under different models implied by alternative theories. A good design will provide probative evidence about which model is correct no matter the underlying state. A poor design might only affirm one model when it is true but fail to provide support for an alternative when *it* is true.
+A separate goal is assessing the performance of a research design under different models implied by alternative theories. A good design will provide probative evidence no matter the treue event generating process. A poor design might provide reliable answers only for specific types of event generating processes.
 
-An important example is assessing the performance of a research design under a "null model" where the true effect size is zero. A good research design should report with a high probability that there is insufficient evidence to reject a null effect. That same research design, under an alternative model with a large effect size, should with a high probability return evidence rejecting the null hypothesis of zero effect. The example makes clear that in order to understand whether the research design is strong, we need to understand how it performs under the models implied by alternative theoretical understandings of the world.
+An important example is the performance of a research design under a "null model," where the true effect size is zero. A good research design should report with a high probability that there is insufficient evidence to reject a null effect. That same research design, under an alternative model with a large effect size, should with a high probability return evidence rejecting the null hypothesis of zero effect. The example makes clear that in order to understand whether the research design is strong, we need to understand how it performs under different models.
 
 ## Summary
 
-If this section left you spinning from the array of choices we have to make in declaring a model, in some ways that was our goal. Inside every power calculator and bespoke design simulation code is an array of assumptions. Some crucially determine design quality and others are unimportant. The salve to the dizziness is in @exm-designagnostically: Design agnostically. Where you are uncertain, explore whether both options produce the same diagnosands. The goal is for our data and answer strategies to hold up under many models.
+If this section left you spinning from the array of choices we have to make in declaring a model, in some ways that was our goal. Inside every power calculator and bespoke design simulation code is an array of assumptions. Some crucially determine design quality and others are unimportant. The salve to the dizziness is in Principle @exm-designagnostically: *Design agnostically*. Where you are uncertain, explore whether both options produce similar diagnosands. The goal is for our data and answer strategies to hold up under many models.
```

Comment:
Here you don't need to write Principle, it can just be 

dizziness is in @exm-designagnostically

For any reference, you don't need to include the word Figure or Table or in this case Principle. You can see how it looks before the change here: https://book.declaredesign.org/declaration-diagnosis-redesign/specifying-model.html